src/logic/Ware.config.ts

```diff
@@ -5,10 +5,14 @@ export const wareConfig = {
         [WareType.Furs]: 355,
         [WareType.Juwelry]: 670,
         [WareType.Salt]: 48,
+        [WareType.Fish]: 10,
```

Comment:
The tests assume that `maxPrice` is strictly larger than `minprice`. They also failed on Travis. I suppose you forgot to run the tests?

Reply:
same for beer

Reply:
I fixed the price boundaries and the tests have all passed. Is there an easy way to fix the ''two issues in one pull request''? I overlooked it...